lib/log-entry.js

```diff
@@ -0,0 +1,8 @@
+export default function(
+    logDate, logMessage, level, parent, leaves) {
+  this.logDate = logDate
+  this.logMessage = logMessage
+  this.level = level
+  this.parent = parent

```

Comment:
apparently "parent" is a reserved variable on the client, while it's a server-side code I'd recommend to use `parentNode`